modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -238,7 +252,7 @@ class Configuration(object):
         if key:
             prefix = self.api_key_prefix.get(identifier)
             if prefix:
-                return "%s %s" % (prefix, key)
+                return "%s%s" % (prefix, key)
```

Comment:
What about keeping the space in between?

Users upgrading to a new version with this change will encounter issues as the space is now gone.

Reply:
True. I think to address this I need to introduce one more configuration parameter that specifies whether there should be a space or not. This is more complicated to configure for the end user but at least it retains the backward compatibility.

Reply:
Removing the space and storing the space in the prefix would be a general solution. But adding a new spacer parameter lets the user control it and ia a backwards compatible fix. Then you could use
%s%s%s (prefix, spacer, key)
where we default spacer to " " if it was not set in api_key or api_key_prefix

Reply:
What about targetting this fix for 4.3.0 (upcoming minor release - breaking changes with fallback)?

I personally prefer not to introduce another parameter as too many options in the configuration is not a good thing.

Is it correct to say that the space is causing an issue in your use case?

Reply:
If we do that could we also refactor to remove the api_key_prefix dict mentioned above? All this info is about the api key value pair, it is unnecessarily complex to use two dicts to store it.

Reply:
> What about targetting this fix for 4.3.0 (upcoming minor release - breaking changes with fallback)?
> 
> I personally prefer not to introduce another parameter as too many options in the configuration is not a good thing.
> 
> Is it correct to say that the space is causing an issue in your use case?
> 
> If it's not causing the issue, then I suggest we leave it as it's to make it consistent with other clients (e.g. ruby, php, etc) and revisit when someone has such a use case.

Yes, the space is causing a problem in my case. Naively it makes sense to automatically add a space when the generated auth is "PREFIX KEY". But what about the case when the generated auth must be "FOO=BAR" (which is my case)? In that case you need to set the prefix to "FOO=", and there is no reason why the generated auth would become "FOO= BAR" with a space in between.

Reply:
> If we do that could we also refactor to remove the api_key_prefix dict mentioned above? All this info is about the api key value pair, it is unnecessarily complex to use two dicts to store it.

Sure, I can do that.

Reply:
> But what about the case when the generated auth must be "FOO=BAR"

A workaround is to set the API key directly as "FOO=BAR" without using the prefix.

Reply:
I have committed a fix that should address the above comments. Let me know if this works for you.